docs/markdown/Reference-manual.md

```diff
@@ -471,6 +471,21 @@ Installs the specified man files from the source tree into system's man director
 
 Installs the entire given subdirectory and its contents from the source tree to the location specified by the keyword argument `install_dir`. Note that due to implementation issues this command deletes the entire target dir before copying the files, so you should never use `install_subdir` to install into two overlapping directories (such as `foo` and `foo/bar`) because if you do the behavior is undefined.
 
+### install_link()
+
+``` meson
+    ctarget install_link(source, destination)
+```
+
+Installs a target as a hard or symbolic. Symlinks can only be created from installed targets, while hard links can be created from both targets regardless of whether they're installed or not. It will preserve the file extension of the source file, if it has one.
+
+It additionally supports the following keyword arguments:
+
+ - `install_dir`, the location to install to, this keyword is required.
```

Comment:
I think if `install_dir:` is omitted, the same install directory as the target should be used.